### PR TITLE
Star Wars - Droids Compatibility Patch

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Patches: Star Wars - Droids Core (Continued), Star Wars - Astromech Droids (Continued), Star Wars - Protocol Droids (Continued) and Star Wars - Support Droids (Continued)

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+							<power>10</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>Hands</linkedBodyPartsGroup>
+							<armorPenetrationSharp>1</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.250</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>Electric Arc Welder</label>
+							<capacities>
+								<li>Cut</li>
```
Comment (N7Huntsman) [inline Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml:213]: I'd switch this weapon to just Burn damage.

File: Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+					<MeleeCritChance>1</MeleeCritChance>
+					<MeleeParryChance>0</MeleeParryChance>
+					<SmokeSensitivity>0</SmokeSensitivity>
+					<!--<Suppressability>0</Suppressability>-->
+					<CarryBulk>30</CarryBulk>
+				</value>
+			</li>
+			
+				
+			<li Class="PatchOperationReplace">
```
Comment (N7Huntsman) [inline Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml:179]: I'd half the damage of all of these attacks.

File: Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+							<cooldownTime>1.8</cooldownTime>
+							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.625</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
```
Comment (N7Huntsman) [inline Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml:237]: Again, would half all of these damages.

File: Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml [hunk 4]
```diff
@@ -0,0 +1,476 @@
...
+				<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="R4_Astromech"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
```
Comment (N7Huntsman) [inline Patches/Star Wars - Droids/Patch_SWDroids_Astromech.xml:271]: Half the damage.

File: Patches/Star Wars - Droids/Patch_SWDroids_Core.xml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+		<li>Star Wars - Droids Core (Continued)</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			
+			
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[
```
Comment (N7Huntsman) [inline Patches/Star Wars - Droids/Patch_SWDroids_Core.xml:15]: It seems like a few of those various attacks should have different damages; Cut for some of the saws, Burn for the welder, Stab for the injector, Blunt for the arm, etc.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Some tweaks to be made.

File: Patches/Star Wars - Droids/Patch_SWDroids_Core.xml (hunk 5 above)
Comment (Person243546) [reply Patches/Star Wars - Droids/Patch_SWDroids_Core.xml:15]: I think these are placeholders. When installed on the droids it should use their melee attacks instead.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**